skbeam/core/accumulators/projector.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+
+"""
+authors: Mikhail Dubrovin, TJ Lane, Christopher O'Grady
+"""
+
+
+class Projector(object):
+
+    def __init__(self, bin_values, nbins, weights=None, norm=True):
```

Comment:
This should take `vmin` and `vmax` as inputs instead of always doing 'tight' binning.  Matching the `np.histogram` api would be a good idea.